src/treeChooser/treeChooser.directive.js

```diff
@@ -127,6 +127,22 @@ function TreeChooser($timeout) {
         event.stopPropagation();
       });
 
+      var treeChooserDiv = angular.element(element[0].querySelector('.treeChooser'));
```

Comment:
Can't all this logic just be with ng-attr-tabindex (or something similar)?

Reply:
I'm receiving an error:
TypeError: treeChooserDiv.focus is not a function
    at Scope.scope.focusTreeChooserDiv (tree-chooser.js:219)
    at TreeChooserController.vm.setSelected (tree-chooser.js:627)
    at TreeChooserController.vm.toggleSelected (tree-chooser.js:583)
    at TreeChooserController.vm.toggleSelectedActive (tree-chooser.js:562)
    at TreeChooserController.vm.onListKeyDown (tree-chooser.js:365)
    at fn (eval at compile (angular.js:15126), <anonymous>:4:367)

Reply:
That's the wrong approach. It may work on your machine due to differences in jquery. But you should just follow the same approach as above: treeChooserDiv[0].focus. If this doesn't work, I may send an email about what I think will work the best. I thought about it last night

Reply:
understand you. changed the code